docker-compose-test.yml

```diff
@@ -1,16 +1,21 @@
 services:
-  mongo:
+  mongodb:
     image: mongo:5.0
     ports:
       - 27017:27017
+    expose:
+      - 27017
```

Comment:
Pas certaine que cette ligne soit utile si ?
https://docs.docker.com/engine/reference/builder/#expose
"The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published."